Chiazokam/Politico
# #163351844 create the get one party endpoint

#### What does this PR do?
This PR tests and creates the GET one parties endpoint

#### Description of Task to be completed?
- write tests for all the functions
- create the endpoint

#### How should this be manually tested?
- Clone the repo - `https://github.com/Chiazokam/Politico`
- Enter into the project directory - `cd Politico`
- switch to this branch - `checkout ft-datastructures-get-one-party-163351844`
- Start the server `npm run start:dev`
- Open `Postman`
- On `postman` Make a `GET` request to `http://localhost:3000/api/v1/parties/1 `
- You should receive a successful response

#### Any background context you want to provide?
Nil
#### What are the relevant pivotal tracker stories?
[#163351844](https://www.pivotaltracker.com/story/show/163351844)

####  Images

![image](https://user-images.githubusercontent.com/26940294/51780130-5d4a8600-210c-11e9-91ef-ee80ea2db13c.png)
